presto-thrift-connector/src/main/java/com/facebook/presto/connector/thrift/util/RetryDriver.java

```diff
@@ -278,9 +276,9 @@ void nextAttempt()
         private final CounterStat success = new CounterStat();
         private final CounterStat finalFailure = new CounterStat();
         private final DistributionStat attemptsBeforeFailure = new DistributionStat();
-        private final TimeDistribution failureLatency = new TimeDistribution(MICROSECONDS);
+        private final DistributionStat failureLatencyMillis = new DistributionStat();
         private final DistributionStat attemptsBeforeSuccess = new DistributionStat();
-        private final TimeDistribution successLatency = new TimeDistribution(MICROSECONDS);
+        private final DistributionStat successLatencyMillis = new DistributionStat();
```

Comment:
You should use TimeStat, not DistributionStat. The former allows you to specify the scale to use when reporting the values, just like TimeDistribution.